lib/active_admin_import/model.rb

```diff
@@ -85,7 +87,7 @@ def encode_file
 
     def uncompress_file
       Zip::File.open(file_path) do |zip_file|
-        self.file = Tempfile.new("active-admin-import-unzipped")
+        self.file = Tempfile.new('active-admin-import-unzipped')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.